### ISSUE TITLE
Could not find a declaration file for module '@devvie/bottom-sheet'

### ISSUE THREAD

**Issue by ErickPetru** [stanleyugwu/react-native-bottom-sheet#30]
We tried to upgrade the package to the latest [0.4.2 release](https://github.com/stanleyugwu/react-native-bottom-sheet/releases/tag/v0.4.2), since we need the https://github.com/stanleyugwu/react-native-bottom-sheet/pull/28 fix for our React Native for Web based project.

But after installing it, TypeScript complains there's no types from this package:

![image](https://github.com/user-attachments/assets/4b48bfb2-e1fa-4a35-94e5-56018ec42eb0)

We checked the installed package folder and confirmed there's no `index.d.ts` file, while it was there in previous the 0.3.0 version.

**Comment by stanleyugwu** [comment]
Likely `tsc` compilation issue, thanks for pointing this out; I'll look into it asap. For now maybe just `@ts-ignore` it.

**Comment by mrkarimi2001** [comment]
not solved?

**Comment by vickyRathee** [comment]
Still the same error in `0.4.2`

**Comment by thejoaov** [comment]
@stanleyugwu @vickyRathee i opened a pr for that, meanwhile, you can use this patch

```diff
diff --git a/package.json b/package.json
index 56f36457010d9a7c0583f7fd9ba00dad2351e189..9680ddc6841d5e18194bba7193a3e3a8fcaacfdb 100644
--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "description": "The 😎smart , 📦tiny , and 🎗flexible bottom sheet your app craves 🚀",
   "main": "lib/commonjs/index",
   "module": "lib/module/index",
-  "types": "lib/typescript/index.d.ts",
+  "types": "lib/typescript/src/index.d.ts",
   "react-native": "src/index",
   "source": "src/index",
   "files": [
```

**Comment by stanleyugwu** [comment]
Hey y'all, I just released version `0.4.3` which has the fix for this.

@thejoaov thanks again for making this fix
